modules/public/static.go

```diff
@@ -18,8 +18,19 @@ import (
 	"time"
 
 	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/util"
 )
 
+// GlobalModTime provide a gloabl mod time for embedded asset files
+func GlobalModTime(filename string) time.Time {
+	t, err := util.GetExecutableModTime()
+	if err != nil {
+		log.Error("GetExecutableModTime failed:", err)
+		return time.Now()
```

Comment:
time.Now() seems not good, it will make all files non-cacheable.

At least we could use the start time of this Gitea process.